config/game_options.txt

```diff
@@ -446,13 +446,13 @@ ROUNDSTART_RACES ipc
 ROUNDSTART_RACES ethereal
 ROUNDSTART_RACES apid
 #ROUNDSTART_RACES jelly
-#ROUNDSTART_RACES golem
-#ROUNDSTART_RACES adamantine
-#ROUNDSTART_RACES plasma
-#ROUNDSTART_RACES diamond
-#ROUNDSTART_RACES gold
-#ROUNDSTART_RACES silver
-#ROUNDSTART_RACES uranium
+#ROUNDSTART_RACES iron_golem
```

Comment:
You need to edit the game options for Sage as well.

Reply:
done, thanks for reminding me